test/fuzz/test_runner.py

```diff
@@ -194,7 +203,7 @@ def job(command):
             "-runs=100000",
             target_seed_dir,
         ]
-        futures.append(fuzz_pool.submit(job, command))
+        futures.append(fuzz_pool.submit(job, command, target))
```

Comment:
Command here is still the old binary path.

Reply:
nice catch. Fixed